swagger/openemr-api.yaml

```diff
@@ -6586,6 +6586,41 @@ paths:
       security:
         -
           openemr_auth: []
+  /fhir/ValueSet:
+    get:
+      tags:
+        - fhir
+      description: 'Returns a list of ValueSet resources.'
+      parameters:
+        -
+          name: _id
+          in: query
+          description: 'The uuid for the ValueSet resource.'
+          required: false
+          schema:
+            type: string
+      responses:
+        '200':
+          description: 'Standard Response'
+          content:
+            application/json:
+              schema:
+                properties:
+                  'json object': { description: 'FHIR Json object.', type: object }
+                type: object
+                example:
+                  meta: { lastUpdated: '2021-09-14T09:13:51' }
+                  resourceType: Bundle
+                  type: collection
+                  total: 0
+                  link: [{ relation: self, url: 'https://localhost:9300/apis/default/fhir/ValueSet' }]
+        '400':
+          $ref: '#/components/responses/badrequest'
+        '401':
+          $ref: '#/components/responses/unauthorized'
+      security:
+        -
+          openemr_auth: []
```

Comment:
looks like this is missing the /fhir/ValueSet/{uuid} endpoint
Note this script is built (we do not manually edit it) via a script. Let me know if need guidance on doing that.

Reply:
The syntax for examples didn't like array [], replaced w {} and it worked. Thanks!